src/opa.js

```diff
@@ -51,8 +72,14 @@ function _dumpJSON(wasmInstance, memory, addr) {
 
 const builtinFuncs = builtIns;
 
-// _builtinCall dispatches the built-in function. The built-in function
-// arguments are loaded from Wasm and back in using JSOn serialization.
+/**
+ * _builtinCall dispatches the built-in function. The built-in function
+ * arguments are loaded from Wasm and back in using JSON serialization.
+ * @param {OpaInstance} wasmInstance 
+ * @param {WebAssembly.Memory} memory 
+ * @param {typeof builtIns} builtins
```

Comment:
I'm a little bit confused by this syntax, this is saying the parameter builtins has the type of the global `builtins` variable?

We really should rename some of this stuff.. but I think that as-is the parameter value is a map of id numbers to function name strings.